sdk/src/cameras/ad-fxtof1-ebz/camera_fxtof1.cpp

```diff
@@ -381,12 +382,12 @@ aditof::Status CameraFxTof1::requestFrame(aditof::Frame *frame,
     uint16_t *frameDataLocation;
     frame->getData(FrameDataType::FULL_DATA, &frameDataLocation);
 
-    status = m_depthSensor->getFrame(frameDataLocation);
+    status = m_depthSensor->getFrame(frameDataLocation, bufferInfo);
     if (status != Status::OK) {
         LOG(WARNING) << "Failed to get frame from device";
         return status;
     }
-
+    LOG(INFO) << "TIMESTAMP: " << bufferInfo->timestamp;
```

Comment:
It would be too noisy to print timestamp for each frame while streaming a video.

Reply:
It is just for test purposes, will be removed.